packages/core/src/components/skeleton/_skeleton.scss

```diff
@@ -0,0 +1,111 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+
+.pt-skeleton-container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+
+@mixin bone-width($width) {
+  &.pt-skeleton-bone-#{$width} {
+    width: $width * 1%;
+  }
+}
+
+.pt-skeleton-bone {
```

Comment:
i'm pretty sure this is _all_ we need for the minimal CSS API. don't want to impose any larger structure on consumers of the CSS API. 

they should be able to simply slap a single class on any element to change its appearance to a loading bone. then width may have to be adjusted in CSS for optimal appearance.